examples/terraform-submodules/gke-local/module.tf

```diff
@@ -24,30 +24,43 @@ provider "google-beta" {
   version = "~> 2.10"
 }
 
+variable "project" {
+  default = ""
+}
+
+variable "name" {
+  default = "agones-terraform-example"
+}
+
 // Install latest version of agones
 variable "agones_version" {
   default = ""
 }
 
-// Your GKE project name
-variable "project" {
-  default = "agones"
+variable "machine_type" {
+  default = "n1-standard-4"
 }
-module "gke_cluster" {
 
+// Note: This is the number of gameserver nodes. The Agones module will automatically create an additional
+// two node pools with 1 node each for "agones-system" and "agones-metrics".
+variable "node_count" {
+  default = "4"
+}
+
+module "gke_cluster" {
   source = "../../../install/terraform/modules/gke"
 
   cluster = {
-    "project"          = var.project
     "zone"             = "us-west1-c"
-    "name"             = "test-cluster3"
-    "machineType"      = "n1-standard-4"
-    "initialNodeCount" = "4"
+    "name"             = var.name
+    "machineType"      = var.machine_type
+    "initialNodeCount" = var.node_count
+    "project"          = var.project
+    "network"          = "default"
   }
 }
 
 module "helm_agones" {
-
   source = "../../../install/terraform/modules/helm"
 
   agones_version         = var.agones_version
```

Comment:
Next line:
```
values_file            = ""
```
Should be changed to 
```
values_file            = var.values_file
``` 
and please create a new variable values_file
It should contain 
`-var values_file="../helm/agones/values.yaml"`
when we install from current branch, not release master.

Reply:
I'm not sure I understand why we want to do this?

Do we want to be sending users to look into our helm files? Seems like something they shouldn't need or want to do?

Reply:
This is needed for developer tooling, we can fix this after 1.4, but what this is optional:
1. there could be values from the helm chart;
2. For development we should have an option to change something to values file and it would be reflected in the deployment